server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -761,9 +764,8 @@ func (e *EventChannel) processSingleEvent(event *inpb.InvocationEvent, iid strin
 		if e.logWriter != nil {
 			e.logWriter.Write(e.ctx, []byte(p.Progress.Stderr))
 			e.logWriter.Write(e.ctx, []byte(p.Progress.Stdout))
-			// For now, write logs to both chunks and the invocation proto
-			// p.Progress.Stderr = ""
-			// p.Progress.Stdout = ""
+			p.Progress.Stderr = ""
+			p.Progress.Stdout = ""
```

Comment:
nit: It looks like we do this in event_parser.go as well -- would it be fine to remove it from here?

Reply:
No, we need both: this one prevents us from writing the logs into the event proto stream, while the other call stops us from transmitting a console buffer for older invocations that have both a chunked log and a console buffer.

Reply:
sg -- mind adding a comment as to why these are here?

Reply:
Done!